samples/Hystrix.Dotnet.Samples.AspNetCore/Controllers/HelloController.cs

```diff
@@ -19,10 +19,19 @@ public HelloController(IHystrixCommandFactory hystrixCommandFactory)
         
         [HttpGet]
         public async Task<IActionResult> Get()
-        { 
+        {
+            var firstCall = true;
+
             var result = await hystrixCommand.ExecuteAsync(
                 async () =>
                 {
+                    if(firstCall)
+                    {
+                        // On the first call we throw an exception to be able to illustrate Retry.
+                        firstCall = false;
+                        throw new InvalidOperationException();
```

Comment:
Perhaps if you can set a list of exception types to retry on and set only a network exception as default?